patcher/src/ts/sidebar/ServerSelect.tsx

```diff
@@ -62,6 +63,13 @@ class SelectServer extends React.Component<SelectServerProps, SelectServerState>
     )
   }
 
+  trySelectCharacter = (shardID:number) => {
+    if(!this.props.charactersState || !this.props.charactersState.characters || 0 == this.props.charactersState.characters.length) return;
+
+    let filteredCharacters = this.props.charactersState.characters.filter( c => c.shardID == shardID);
```

Comment:
const not let
=== not ==
https://github.com/airbnb/javascript#comparison--eqeqeq
https://github.com/airbnb/javascript#references--prefer-const